packages/legend-graph/src/graphManager/protocol/pure/v1/engine/analytics/V1_StoreEntitlementAnalysis.ts

```diff
@@ -116,6 +116,27 @@ const V1_deserializeDatasetSpecification = (
     }
   }
 };
+export class V1_EntitlementReportAnalyticsInput {
+  storeEntitlementAnalyticsInput!: V1_StoreEntitlementAnalysisInput;
+  reports: V1_DatasetSpecification[] = [];
+
+  static readonly serialization = new SerializationFactory(
+    createModelSchema(V1_EntitlementReportAnalyticsInput, {
+      storeEntitlementAnalyticsInput: optionalCustom(
```

Comment:
why `optionalCustom`, this should be `usingModelSchema(V1_StoreEntitlementAnalysisInput.serialization.schema)`